public/app/core/components/Login/LoginServiceButtons.tsx

```diff
@@ -15,6 +15,10 @@ const loginServices: () => LoginServices = () => {
       enabled: oauthEnabled && config.oauth.google,
       name: 'Google',
     },
+    azuread: {
+      enabled: config.oauth.azuread,
+      name: 'Microsoft',
```

Comment:
I'm not 100% sure about button label. @torkelo @daniellee @xlson what do you think - which name should we use - Microsoft, Azure, Azure AD, Microsoft Azure, etc?

Reply:
I name it Microsoft due to this guide https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-add-branding-in-azure-ad-apps I also got the icon from there.

Reply:
Ok, `Sign in with Microsoft` looks good thus.
>DON’T expose end users to the Azure or Active Directory brands. It’s ok however to use these terms with developers, IT pros, and admins.

The only question are all Grafana users _developers, IT pros, and admins_ so we can use Azure name :)
But anyway, from the Microsoft branding guide `Sign in with Microsoft` looks better and quite unambiguous.

Reply:
`Sign in with Microsoft` is in line with the rest of the ui. We already have `Sign in with Google` and `Sign in with Grafana.com`